Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,76 @@
+
+#### Integrations
+
+##### CrowdStrike Falcon
+
+- Deprecated the ***cs-falcon-get-behavior*** Command, no replacement available.
+- Added support for alignment with the Raptor release, which will become the default and cause a breaking change. A checkbox has been added for older versions.
+- In Command ***cs-falcon-search-detection*** the following outputs will no longer exist in the default version:
```

Comment:
How this will impact the current PBs?
`id: CrowdStrike Falcon - Get Detections by Incident`
`id: CrowdStrike Falcon - SIEM ingestion Get Incident Data`